static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -298,13 +311,21 @@ class UsageStatsOrganization<
         ),
         score: filtered,
       },
-      dropped: {
-        title: tct('Dropped [dataCategory]', {dataCategory: dataCategoryName}),
+      rateLimited: {
+        title: tct('Rate Limited [dataCategory]', {dataCategory: dataCategoryName}),
+        help: tct(
+          'Rate Limited [dataCategory] were discarded due to rate-limits, abuse, or cardinality limits',
+          {dataCategory}
+        ),
+        score: rateLimited,
+      },
+      invalid: {
+        title: tct('Invalid [dataCategory]', {dataCategory: dataCategoryName}),
         help: tct(
-          'Dropped [dataCategory] were discarded due to invalid data, rate-limits, quota limits, or spike protection',
+          'Invalid [dataCategory] were sent by the SDK and were discarded because the data did not meet the basic schema requirements',
```

Comment:
@jan-auer can you help me with the description here...is it good enough?

Reply:
This is good, thank you!